### PR TITLE
Feature/unicode support

### LINKED ISSUE #417: eCAL Player cannot load measurements from paths with non-ASCII characters
**Describe the bug**
The eCAL Player fails to load measurements from paths with non-ASCII characters, at least on Windows.

**Screenshots**
grafik

**Additional context**
I should probably also check whether the **eCAL Recorder** can handle non-ascii characters

### PR DESCRIPTION
**Pull request by FlorianReimold**
Fixes #417 

Changed hopefully all Windows calls to ASCII-fuctions (like `CreateDirectoryA` or `open()`) by WideString variants. eCAL will from now on assume everything to be UTF-8 encoded, which should make perfect sense in 2022.

### CODE REVIEW

**Review by rex-schilasky — APPROVED** [review]
So far okay from my side. It's a large PR, so just a rough review file by file ..

File: contrib/ecalhdf5/CMakeLists.txt [hunk 1]
```diff
@@ -43,7 +43,6 @@ set(ecalhdf5_src
     src/eh5_meas_file_v5.cpp
     src/eh5_meas_file_v5.h
     src/eh5_meas_impl.h
-    src/eh5_util.h
     src/escape.cpp
```
Comment (Kerstin-Keller) [inline contrib/ecalhdf5/CMakeLists.txt:46]: does this line have anything to do with the rest?

**Review by Kerstin-Keller — COMMENTED** [review]
Except my comment I don't have any remarks.
Except that I hate all those `#ifdef WIN32`, though I don't think that there is a solution for that.

File: contrib/ecalhdf5/CMakeLists.txt (hunk 1 above)
Comment (FlorianReimold) [reply contrib/ecalhdf5/CMakeLists.txt:46]: The eh5-util.h was a utily header that wasn't used any more. So I didn't bother with unicode-porting and just removed it. It all uses the EcalUtils now, which supports unicode paths.

**Pull request merged**